src/rpc/blockchain.cpp

```diff
@@ -284,6 +287,10 @@ static UniValue getchainalgostats(const JSONRPCRequest& request)
     }
 
     UniValue obj(UniValue::VOBJ);
+
+    obj.pushKV("start", (uint64_t)pindex->nTime);
+    obj.pushKV("finish", (uint64_t)chainActive.Tip()->nTime);
+    obj.pushKV("period", (uint64_t)(chainActive.Tip()->nTime - pindex->nTime)/60);
```

Comment:
NIT now I really want start and end block. It would make a graph I want to build for veil-stats much easier.